examples/dtnerouting.rs

```diff
@@ -0,0 +1,293 @@
+use anyhow::{bail, Result};
+use clap::{crate_authors, crate_version, App, Arg};
+use dtn7::routing::erouting::ws_client::{new, Command};
+use dtn7::routing::erouting::{Packet, Sender, SenderForBundleResponse};
+use dtn7::DtnPeer;
+use futures::channel::mpsc::unbounded;
+use futures_util::{future, pin_mut, StreamExt};
+use log::{debug, info};
+use std::collections::{HashMap, HashSet};
+
+fn epi_add(history: &mut HashMap<String, HashSet<String>>, bundle_id: String, node_name: String) {
```

Comment:
have an epicore struct and all functions be impl of that as the history is always passed as first param and the epi_ prefix can then be omitted